R/measures.R

```diff
@@ -64,8 +68,13 @@
 #' }
 riem_measures <- function(
     station = "VOHY",
-    date_start = "2014-01-01",
-    date_end = as.character(Sys.Date())) {
+    data = "all",
```

Comment:
the new parameter should not be added before the previous ones, because users might rely on argument order rather than argument names in their calls.

Reply:
Yes, sorry, I messed that up.  I'll rectify

Reply:
Hey, no worries, that's what PR reviews are for, thanks again for contributing!